.jazzy.yaml

```diff
@@ -1,4 +1,4 @@
-module: Cauli
+module: Cauliframework
```

Comment:
What is going on with this diff here? Somehow ... it thinks that this is a change compared to `develop` ... but it isn't.

Reply:
Interesting. I think I found the reason and it isn't a bug (that's what GitHub is telling us): https://github.community/t5/How-to-use-Git-and-GitHub/GitHub-pull-requests-showing-invalid-diff-for-already-merged/td-p/3000

Reply:
And some more insight on this topic here: https://help.github.com/articles/about-comparing-branches-in-pull-requests/#three-dot-and-two-dot-git-diff-comparisons

Reply:
Alright. I think I fixed this for you @Shukuyen. What have I learned? GitHub might do weird things if you rebase.